test/e2e/virtual/workspaces/virtual_workspace_test.go

```diff
@@ -183,6 +185,8 @@ var testCases = []struct {
 		work: func(ctx context.Context, t *testing.T, server runningServer) {
 			testData := newTestData()
 
+			user1Client := server.UserKcpClients[0]
```

Comment:
does `user1Client` exercise redirection code when the vw is run in a standalone mode?